.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  tests:
+    name: Test Compilation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@3cf7f8cc28d1b4e7d01e3783be10a97d55d483c8
+      - run: cargo test --release --lib
+      - run: cargo test --release
```

Comment:
I think the latter is enough, it runs all tests

Reply:
I've fixed it in the other PR